parts/linux/cloud-init/artifacts/components.json

```diff
@@ -1,5 +1,7 @@
+// This is not a valid json. Just for schema discussion
 {
   "ContainerImages": [
+    //old
```

Comment:
Could you share an example based on current containerd components.json? I'm trying to understand what the new schema would look like for a component which has multiple different versions based on the SKU

Reply:
Updated in the code. Please update the latest commit.
Now I start thinking we should probably leave 2 groups `ContainerImages` and `Packages` because
- their schema are quite different
- avoid breaking older images. (We will be just adding new keys but not remove/modify them)